src/jdk.jpackage/share/native/common/app.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2020, 2023 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Missing "," after 2023. Should be: "Copyright (c) 2020, 2023, Oracle and/or its affiliates.".

Reply:
Thank you!!!! Fixed.